raptorWeb/templates/raptormc/applications/appsuccess.html

```diff
@@ -14,9 +14,9 @@
 {% block main_block %}
 <section id="welcome" class="container">
 
-  <header class="card bg-dark opacity-75 text-white p-4 mt-4">
-    <h3 class="display-4 fw-bold text-center">Application Successfully Submitted!</h3>
-    <p>
+  <header class="card bg-dark opacity-75 text-white mt-4">
```

Comment:
I've also modified this page, it is a template that is sent directly to an application's page after a user successfully submitted an app, rather than through a view. I am likely going to trash this in favor of a more generic approach to redirection after app submission.